pkg/discovery/worker/compliance/podaffinity/processor.go

```diff
@@ -101,11 +107,12 @@ func GetNamespaceLabelsSnapshot(ns string, nsLister NamespaceLister) (nsLabels l
 
 // ProcessAffinities returns a map of nodes to list of pods which can be placed on each
 // respective node and a set of pods which have affinities.
```

Comment:
@irfanurrehman  Could you please add some explanation here to tell what the newly added 2 `sets.String` in the returned results are used for?

Reply:
Updated! Thanks for taking a look.